requirements/requirements.txt

```diff
@@ -0,0 +1,8 @@
+cookiecutter
```

Comment:
Let's rename this file to `conda.txt` and only leave `cookiecutter` and `entrypoints` on it.

Also, please create a new file called `tests.txt` and add there `pytest` and `pytest-cookies`.